src/Mod/CAM/PathSimulator/AppGL/SimShapes.h

```diff
@@ -37,6 +37,9 @@
         var[idx++] = z;                                                                            \
     }
 
+// #define SET_TRIPLE(var, idx, x, y, z) {var[idx] = x; var[idx+1] = y; var[idx+2] = z;}
+// #define SET_TRIPLE_OFFS(var, idx, offs, x, y, z) {var[idx] = x + offs; var[idx+1] = y + offs;
+// var[idx+2] = z + offs;}
```

Comment:
We should avoid commented out code. It can always be reverted using git history - this is why git exists in the first place. If the code is only temporarily commented out it should have comment explaining why it was commented, when it should be uncommented and when it is safe to remove it.